### PR TITLE
etc/restore-osc-bindings.conf: add file to restore old osc bindings

### PR DESCRIPTION
**Pull request by na-na-hi**
The OSC changed to use select.lua for various tasks, resulting in a significant change in user interaction. This adds a config file which records OSC behavior changes so that the old behavior can be restored.

Fixes: https://github.com/mpv-player/mpv/issues/15176

### CODE REVIEW

**Review by guidocella — APPROVED** [review]

File: meson.build [hunk 1]
```diff
@@ -1779,7 +1779,8 @@ if get_option('cplayer')
     confdir = get_option('sysconfdir')
 
     conf_files = ['etc/mpv.conf', 'etc/input.conf',
-                  'etc/mplayer-input.conf', 'etc/restore-old-bindings.conf']
+                  'etc/mplayer-input.conf', 'etc/restore-old-bindings.conf',
+                  'etc/restore-osc-bindings.conf']
```
Comment (llyyr) [inline meson.build:1783]: Should this install to doc directly or a script-opts directory inside doc?

**Comment by kasper93** [comment]
@Akemi @sfan5 @Dudemanguy Are you ok with installing new file? Not sure about precedence here.

**Comment by sfan5** [comment]
Sounds okay given we do it for other files.

**Comment by Dudemanguy** [comment]
Yeah no biggie.

**Pull request merged**